core/modules/qproc/IndexMap.cc

```diff
@@ -215,10 +225,14 @@ ChunkSpecVector IndexMap::getIntersect(query::ConstraintVector const& cv) {
     bool hasRegion = true;
     try {
         indexSpecs = _si->lookup(cv);
+        if (LOG_CHECK_LVL(getLogger(), LOG_LVL_TRACE)) {
```

Comment:
I suppose that testing log level is quicker than calling util::formatable, that's why I would keep this test. Would you want me to test this?